solutions/usaco-991.mdx

```diff
@@ -67,5 +67,51 @@ public class Loan {
 ```
   
 </JavaSection>
+
+<PySection>
+
+```python
+import sys
+import math
+
+sys.stdin = open("loan.in", "r")
+sys.stdout = open("loan.out", "w")
+
+input = sys.stdin.readline
```

Comment:
i commented about this on your other pr...why?

Reply:
ill change it

Reply:
`with`

Reply:
yep